src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -2916,11 +2918,11 @@ public Mode3D get3DLayout() {
 			case "oulf":
 			case "top-bottom (left eye first)":
 				isAnaglyph = false;
-				return Mode3D.OUL;
+				return Mode3D.ABL;
 			case "ourf":
```

Comment:
Sorry, it has never been a MediaInfo string since the beginning AFAIK:
https://github.com/MediaArea/MediaInfoLib/commit/11ca3e72b2f1fad071a0510e6134bf35dd30442b